radix-engine/src/kernel/call_frame.rs

```diff
@@ -159,10 +159,11 @@ impl CallFrame {
         }
 
         for node_id in &temp_references {
-            self.temp_node_refs
-                .entry(node_id.clone())
-                .or_default()
-                .add_assign(1);
+            if let Some(count) = self.temp_node_refs.get_mut(&node_id) {
+                *count += 1;
+            } else {
+                self.temp_node_refs.insert(node_id.clone(), 1);
+            }
```

Comment:
_Note from author:_

Here is one of the places where I had to replace `entry()` with an equivalent.
Please see if it is correct AND acceptable from a performance PoV.
(as an alternative, we can switch to an `IndexMap`)

Reply:
N/A.
_(I had some difficulties with importing `.entry()` the right way, but today I learned rust and did it)_